pipelines/azure_pipelines.yml

```diff
@@ -3,27 +3,139 @@ trigger:
+          inputs:
+            projects: $(Build.SourcesDirectory)\test\E2ETest\Microsoft.Test.E2E.AspNet.OData\Build.AspNetCore3x\Microsoft.Test.E2E.AspNetCore3x.OData.csproj
+            arguments: --configuration $(BuildConfiguration) --no-incremental
+        - task: PowerShell@2
+          displayName: PowerShell Script
+          inputs:
+            filePath: '$(Build.SourcesDirectory)\tools\MeasureCodeSharing.ps1 '
+            arguments: $(Build.SourcesDirectory)
+        - task: PowerShell@2
+          displayName: Skip StrongName for Classic
+          inputs:
+            targetType: inline
+            script: |
+              $PROGRAMFILESX86 = [Environment]::GetFolderPath("ProgramFilesX86")
+              $SN = $PROGRAMFILESX86 + "\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\sn.exe"
+              $SNx64 = $PROGRAMFILESX86 + "\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\sn.exe"
+              & $SN /Vr $(Build.SourcesDirectory)\bin\Release\Microsoft.AspNet.OData.dll
+              & $SNx64 /Vr $(Build.SourcesDirectory)\bin\Release\Microsoft.AspNet.OData.dll
+              & $SN /Vr $(Build.SourcesDirectory)\bin\release\netstandard2.0\Microsoft.AspNetCore.OData.dll
+              & $SNx64 /Vr $(Build.SourcesDirectory)\bin\release\netstandard2.0\Microsoft.AspNetCore.OData.dll
+              & $SN /Vr $(Build.SourcesDirectory)\bin\Release\UnitTest\AspNet\Microsoft.AspNet.OData.Test.dll
+              & $SNx64 /Vr $(Build.SourcesDirectory)\bin\Release\UnitTest\AspNet\Microsoft.AspNet.OData.Test.dll
+              & $SN /Vr $(Build.SourcesDirectory)\bin\Release\E2ETest\AspNet\Microsoft.Test.E2E.AspNet.OData.dll
+              & $SNx64 /Vr $(Build.SourcesDirectory)\bin\Release\E2ETest\AspNet\Microsoft.Test.E2E.AspNet.OData.dll
+              & $SN /Vr $(Build.SourcesDirectory)\bin\release\E2ETest\AspNetCore\Microsoft.Test.E2E.AspNetCore.OData.dll
+              & $SNx64 /Vr $(Build.SourcesDirectory)\bin\release\E2ETest\AspNetCore\Microsoft.Test.E2E.AspNetCore.OData.dll
+        - task: VSTest@2
+          displayName: Classic Unit Tests (Microsoft.AspNet.OData.Test.dll)
+          inputs:
+            testAssemblyVer2: '**\bin\**\UnitTest\**\Microsoft.AspNet.OData.Test.dll'
+        - task: DotNetCoreCLI@2
+          displayName: 'Core Unit Tests (Microsoft.AspNetCore.OData.Test.csproj) '
+          inputs:
+            command: test
+            projects: $(Build.SourcesDirectory)\test\UnitTest\Microsoft.AspNetCore.OData.Test\Microsoft.AspNetCore.OData.Test.csproj
+            arguments: --configuration $(BuildConfiguration) --no-build
+        - task: VSTest@2
+          displayName: Classic E2E Tests (Microsoft.Test.E2E.AspNet.OData.dll)
+          inputs:
+            testAssemblyVer2: '**\bin\**\E2ETest\**\Microsoft.Test.E2E.AspNet.OData.dll'
+            testFiltercriteria: (DisplayName!=Microsoft.Test.E2E.AspNet.OData.ODataPathHandler.UnicodeRouteTests_Todoü.CRUDEntitySetShouldWork)
```

Comment:
Out of scope, but would you know why we filter out this particular test(s)?

Reply:
I'm not sure. My initial guess woulld be to reduce runtime.